### ISSUE TITLE
PyJWT flake in CI

### ISSUE THREAD

**Issue by mk46** [envoyproxy/envoy#15789]
CI fails due to `pyjwt 2.0.1`
```
ERROR: Cannot install -r /source/tools/dependency/requirements.txt (line 1) and PyJWT==2.0.1 because these package versions have conflicting dependencies.
The conflict is caused by:
    The user requested PyJWT==2.0.1
    pygithub 1.54.1 depends on pyjwt<2.0

```

Please see https://dev.azure.com/cncf/envoy/_build/results?buildId=70832&view=logs&jobId=c5dd2866-6ab3-5f3c-3a44-4cef0ec909b5&j=1c04237c-5f52-5959-5f83-5522b3556331&t=db3f3889-c922-566b-fa2e-4bc4948bc467

**Comment by mk46** [comment]
@phlax Would you mind to take a look?

**Comment by phlax** [comment]
this will be fixed in #15781

**Comment by phlax** [comment]
should be fixed now
